components/test_pd_client/src/pd.rs

```diff
@@ -1392,6 +1392,17 @@ impl TestPdClient {
         panic!("region {} must have leader: {:?}", region_id, peer);
     }
 
+    pub fn region_leader_must_not_be(&self, region_id: u64, peer: metapb::Peer) {
```

Comment:
Maybe we can replace `region_leader_must_not_be` and `region_leader_must_be` to a more generate function `check_region_leader` to reduce redundance.

Reply:
done